tests/summmarizer/test_text_media_matching.py

```diff
@@ -0,0 +1,127 @@
+from classifier_and_summarizer.summarizer.text_media_matcher import (
+    ElementWithIndex, TextMediaMatcher)
+
+
```

Comment:
It might be good to add a test for each method verifying that the method actually performs what it is supposed to do. This will make it easier to debug issues later.

Reply:
Will add tests for the same!